sdk/agrifood/Azure.Verticals.AgriFood.Farming/Azure.Verticals.AgriFood.Farming.sln

```diff
@@ -1,16 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29709.97
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33627.172
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Analytics.FarmBeats", "src\Azure.Verticals.AgriFood.Farming.csproj", "{E33D09D9-D809-472C-82E6-6A26BDB86FC2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Verticals.AgriFood.Farming", "src\Azure.Verticals.AgriFood.Farming.csproj", "{E33D09D9-D809-472C-82E6-6A26BDB86FC2}"
```

Comment:
Is this change related to the stress framework?

Reply:
No, this was due to a rename that took place which the solution file never absorbed.  When I removed and rebuilt the solution, VS updated the metadata to match the current project name.